rts/Sim/Weapons/Weapon.cpp

```diff
@@ -1370,6 +1433,8 @@ float CWeapon::GetAccuratePredictedImpactTime(const CUnit* unit) const
 			// cannon has no gravity
 			// non-parabolic projectiles are much easier
 			// simple quadratic equation
+			predictTime = GetSafeInterceptTime(unit, predictMult);
+			/*
```

Comment:
don't leave commented out code, or at least leave a hint on how it may become useful in the future